README.md

```diff
@@ -6,35 +6,133 @@ Exercism exercises in Haskell
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
-
-### Development Dependencies
-
-You should have [Stack](http://docs.haskellstack.org/) installed in your
-system to make contributing to this repository easier.
-
-### Exercises
-
-All exercises have the following structure:
+- [Asking for help](#asking-for-help)
+- [How to contribute](#how-to-contribute)
+  * [Reporting or fixing bugs](#reporting-or-fixing-bugs)
+  * [Reviewing issues and pull requests](#reviewing-issues-and-pull)
+  * [Porting exercises](#porting-exercises)
+  * [Updating an exercise test suite](#updating-an-exercise-test-suite)
+- [Repository structure and conventions](#repository-structure-and-conventions)
+  * [Directory structure](#directory-structure)
+  * [Exercise structure](#exercise-structure)
+- [Writing an issue](#writing-an-issue)
+- [Writing a pull request](#writing-a-pull-request)
+- [Development Dependencies](#development-dependencies)
+- [Stub solution](#stub-solution)
+- [Example solution](#example-solution)
+- [Test suite](#test-suite)
+- [Running tests](#running-tests)
+- [Running HLint](#running-hlint)
+
+### Asking for help
+When in doubt just ask!
+- [discussions](https://github.com/exercism/discussions): general questions.
```

Comment:
I'm afraid that linking the `discussions` repository will generate a lot o noise there.

Gitter and `xhaskell` issues seem the way to go, IHMO.